pyproject.toml

```diff
@@ -162,44 +162,57 @@ check_untyped_defs = true
 
 [tool.coverage.run]
 omit = [
-  # main.py should be checked separately in an integration test.
-  "*/main.py",
-  "*/__init__.py",
-  # This is vendored source code from another project
-  "src/orquestra/sdk/shared/kubernetes/quantity.py",
+    # main.py should be checked separately in an integration test.
+    "*/main.py",
+    "*/__init__.py",
+    # This is vendored source code from another project
+    "src/orquestra/sdk/shared/kubernetes/quantity.py",
 ]
 
 [tool.coverage.report]
 skip_covered = true
 exclude_lines = [
-  "pragma: no cover",
-  "def __repr__",
-  "def __str__",
-  "raise AssertionError",
-  "raise NotImplementedError",
-  "if __name__ == .__main__.:",
-  "assert_never",
+    "pragma: no cover",
+    "def __repr__",
+    "def __str__",
+    "raise AssertionError",
+    "raise NotImplementedError",
+    "if __name__ == .__main__.:",
+    "assert_never",
 ]
 
 [tool.pytest.ini_options]
-log_level="INFO"
+log_level = "INFO"
 
 
 [tool.ruff]
+target-version = "py38"
```

Comment:
Nitpick: I think we can bump that, although im not sure what that would change

Reply:
We technically support 3.8 still, https://github.com/zapata-engineering/orquestra-sdk/pull/386/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R20